packages/lab/src/__tests__/__e2e__/tabs-next/TabsNext.cy.tsx

```diff
@@ -0,0 +1,417 @@
+          });
+        });
+      });
+
+      describe("WHEN the selected tab is clicked", () => {
+        it("THEN focus will be transfered to the selected tab", () => {
+          cy.mount(
+            <StackLayout>
+              <button data-testid="tabstop-1" />
+              <SimpleTabstrip width={400} />
+              <button data-testid="tabstop-2" />
+            </StackLayout>
+          );
+          cy.findByTestId("tabstop-1").focus();
+          cy.findAllByRole("tab").eq(0).realClick();
+          cy.get('[role="tab"]').eq(0).should("be.focused");
+        });
+
+        describe("WHEN the left arrow key is pressed (from first tab)", () => {
+          it("THEN no navigation will occur", () => {
+            cy.mount(<SimpleTabstrip width={400} />);
+            cy.findAllByRole("tab").eq(0).realClick();
+            cy.wait(100); // ArrowRight need some time to move focus after click
+            cy.realPress("ArrowLeft");
+            cy.findAllByRole("tab").eq(0).should("be.focused");
+          });
+        });
+
+        describe("WHEN the right arrow key is pressed", () => {
+          it("THEN focus will be transfered to the next tab", () => {
+            cy.mount(<SimpleTabstrip width={400} />);
+            cy.findAllByRole("tab").eq(0).realClick();
+            cy.wait(100); // ArrowRight need some time to move focus after click
+            cy.realPress("ArrowRight");
+            cy.findAllByRole("tab").eq(1).should("be.focused");
+          });
+        });
+
+        describe("WHEN the tab key is pressed", () => {
+          it("THEN focus will leave the tabstrip", () => {
+            cy.mount(
+              <StackLayout>
+                <button data-testid="tabstop-1" />
+                <SimpleTabstrip width={400} />
+                <button data-testid="tabstop-2" />
+              </StackLayout>
+            );
+            cy.findAllByRole("tab").eq(0).realClick();
+            cy.realPress("ArrowRight");
+            cy.wait(50);
+            cy.realPress("Tab");
+            cy.findByTestId("tabstop-2").should("be.focused");
+          });
+        });
+
+        describe("WHEN focus returns to the tabstrip", () => {
+          it("THEN the selected tab receives focus", () => {
+            cy.mount(
+              <StackLayout>
+                <button data-testid="tabstop-1" />
+                <SimpleTabstrip width={400} />
+                <button data-testid="tabstop-2" />
+              </StackLayout>
+            );
+            cy.findAllByRole("tab").eq(0).should("be.visible");
+            cy.findAllByRole("tab").eq(0).realClick();
+            cy.realPress("ArrowRight");
+            cy.wait(50);
+            cy.realPress("Tab");
+            cy.wait(50);
+            cy.realPress(["Shift", "Tab"]);
+            cy.wait(50);
+            cy.findAllByRole("tab")
+              .eq(0)
+              .should("be.focused")
+              .should("have.ariaSelected");
```

Comment:
how do we know that this was the selected tab? the test only shows that this is. Not totally sure, but I think the second tab should be selected after the ArrowRight (or maybe I'm missing something)